corona-warn-app/cwa-app-ios
# Fix/3246 Correct Flow Logic in Onboarding

## Description
Fixes a bug in the onboarding flow logic. Once the user has enabled logging of contacts then the button "Activate Exposure Logging" is not displayed any more when navigating back and forth in the onboarding process.

### How to Test
1 Onboarding Screen 3 - How to make exposure logging possible: Select "Do Not Activate".
2 on the next screen tap "Back".
3 Onboarding Screen 3: both buttons are displayed. Scroll down to the bottom of the view and check the content of screen 3.
4 Tap on "Activate Exposure Logging" (Screenshot 1).
5 On the next screen tap "Back" (Screenshot 2).
6 Onboarding Screen 3: only one button is displayed (Screenshot 3).
7 Scroll down to the bottom of the view and verify the content of screen 3 is still OK.

## Link to Jira
Story: https://jira.itc.sap.com/browse/EXPOSUREAPP-1692 
Task: https://jira.itc.sap.com/browse/EXPOSUREAPP-3246

## Screenshots
### Screenshot 1
![IMG_0037](https://user-images.githubusercontent.com/7896005/96157096-26d12200-0f12-11eb-8521-2694f3710dfc.PNG)
### Screenshot 2
![IMG_0038](https://user-images.githubusercontent.com/7896005/96157115-2d5f9980-0f12-11eb-9dbf-5d6aec843149.PNG)
### Screenshot 3
![IMG_0039](https://user-images.githubusercontent.com/7896005/96157145-35b7d480-0f12-11eb-8c2d-5c72fb21f640.PNG)
